utils/configcpp/configcpp.h

```diff
@@ -230,8 +230,7 @@ class Config
     xmlDocPtr fDoc;
     const std::string fConfigFile;
     time_t fMtime;
-    mutable boost::mutex fLock;
-    const std::string fInstallDir;
+    mutable boost::recursive_mutex fLock;
```

Comment:
Don't we want to make it std::recursive_mutex?

Reply:
No, because this is used for scoped locks which don't come into std until C++17